components/log-viewer-webui/server/package.json

```
@@ -30,6 +30,7 @@
     "fastify-plugin": "^5.0.1",
     "http-status-codes": "^2.3.0",
     "pino-pretty": "^13.0.0",
+    "socket.io": "^4.0.0",
```

Let's use the latest
```suggestion
    "socket.io": "^4.8.1",
```